pkg/kopia/command/parse_command_output.go

```diff
@@ -327,6 +331,93 @@ func parseKopiaProgressLine(line string, matchOnlyFinished bool) (stats *Snapsho
 	}
 }
 
+// SnapshotRestoreStats is a container for stats parsed from the output of a
+// `kopia snapshot restore` command.
+type SnapshotRestoreStats struct {
```

Comment:
can it just be called `RestoreStats`? i quickly checked what kopia restore command is called and its `kopia restore`  so I think just `RestoreStats` should also be ok.

Reply:
But please feel free to disagree.

Reply:
Actually I was always using `kopia snapshot restore` but I've just checked documentation, and you are right. Seems that `snapshot restore` is an alias to `restore`. So your suggestion makes sense.